src/boot/efi/secure-boot.c

```diff
@@ -35,3 +37,87 @@ static const char sbat[] _used_ _section_(".sbat") =
         SBAT_PROJECT ",1,The systemd Developers," SBAT_PROJECT "," PROJECT_VERSION "," PROJECT_URL "\n"
         SBAT_PROJECT "." SBAT_DISTRO "," STRINGIFY(SBAT_DISTRO_GENERATION) "," SBAT_DISTRO_SUMMARY "," SBAT_DISTRO_PKGNAME "," SBAT_DISTRO_VERSION "," SBAT_DISTRO_URL "\n";
 #endif
+
+EFI_STATUS secure_boot_enroll(EFI_FILE *root_dir, Config *config, ConfigEntry *entry) {
```

Comment:
config is unused, drop it from the function sig. And only `entry->path` is used, just pass it the string directly. (This would even remove the need to move split boot.c entirely.